src/main/java/menagerie/gui/screens/SlideshowScreen.java

```diff
@@ -258,9 +258,17 @@ private void tryDeleteCurrent(boolean deleteFile) {
 
         mediaView.stop();
         if (deleteFile) {
-            new ConfirmationScreen().open(getManager(), "Delete files", "Permanently delete selected files? (1 file)\n\n" + "This action CANNOT be undone (files will be deleted)", onFinish, null);
+            new ConfirmationScreen().open(getManager(), "Delete files",
+                    """
+                            Permanently delete selected files? (1 file)
```

Comment:
I hope this does not introduce additional tabs at the beginning...

Reply:
Just tested it and it still works as intended.